bedrock/mozorg/templates/mozorg/about/governance/policies/social-content-policies.html

```diff
@@ -0,0 +1,639 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "mozorg/about-base.html" %}
+
+{% block page_title %}Mozilla.Social Content Policies{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('social-content-policies') }}
+{% endblock %}
+
+{% block article %}
+<h1 class="mzp-c-article-title">Mozilla.Social Content Policies</h1>
+
+<nav>
+  <ol class="mzp-u-list-styled">
+    <li><a href="#copyright">Copyright</a></li>
+    <li><a href="#trademark">Trademark</a></li>
+    <li><a href="#impersonation">Impersonation</a></li>
+    <li><a href="#harassment">Harassment</a></li>
+    <li><a href="#violent-content">Violent content</a></li>
+    <li><a href="#self-harm">Self-Harm</a></li>
+    <li><a href="#sexualized-content">Sexualized content</a></li>
+    <li><a href="#child-sexual-abuse">Child Sexual Abuse</a></li>
+    <li><a href="#third-party-violations">Violations of Third Party Privacy</a></li>
+    <li><a href="#misinformation">Misinformation</a></li>
+    <li><a href="#illegal-goods">Promotion of illegal goods</a></li>
+    <li><a href="#fraud-counterfeit-goods">Fraud/Counterfeit goods</a></li>
+    <li><a href="#extremist-content">Terrorist and Violent Extremist Content</a></li>
+    <li><a href="#hate-speech">Hate Speech</a></li>
+  </ul>
```

Comment:
Opening is an `<ol>` but closed with a `</ul>`.

Reply:
Nice catch!